packages/core/src/DatumBuilders/DatumBuilder.V3.class.ts

```diff
@@ -311,12 +333,14 @@ export class DatumBuilderV3 implements DatumBuilderAbstract {
       assetB,
     ).schema;
 
+    const feeManagerScript = this.extractFeeManagerScript(feeManager);
```

Comment:
Since `Script` is an actual type of fee manager, let's just name it `getFeeManagerFromAddress` or something similar.